### ISSUE TITLE
No strut in FVWM, please add `_NET_WM_STRUT` window property for backwards compatibility

### ISSUE THREAD

**Issue by joshuakraemer** [rilian-la-te/vala-panel#103]
At the moment, vala-panel sets the `_NET_WM_STRUT_PARTIAL` window property to reserve screen space. Some window managers (e.g. fvwm) do not implement this, but the older `_NET_WM_STRUT`property. Please add this property for compatibility with those window managers.

See https://specifications.freedesktop.org/wm-spec/latest/ar01s05.html#idm139870829953616.

**Comment by rilian-la-te** [comment]
Example of this managers, please. I was not found one when rewriting.

**Comment by joshuakraemer** [comment]
In fvwm, `_NET_WM_STRUT_PARTIAL` is not implemented, see: https://github.com/fvwmorg/fvwm/blob/2e437c9c132d469edfec5e1ed66f227e34978c64/fvwm/ewmh.c#L35.

```
 * Not Implemented (draft 1.3 - 2004-05-01):
 [...]
 * _NET_WM_STRUT_PARTIAL      should be implemented
```

But `_NET_WM_STRUT` is implemented and works as expected, see: https://github.com/fvwmorg/fvwm/blob/2e437c9c132d469edfec5e1ed66f227e34978c64/fvwm/ewmh.c#L223.

**Comment by joshuakraemer** [comment]
Thank you very much, it works. :)
